olclient/schemata/import.schema.json

```
@@ -93,6 +93,18 @@
         "12 ounces",
         "1 pounds"
       ]
+    },
+    "identifiers": {
+      "type": "object",
+      "patternProperties": {
+        "^\\w+": { "$ref": "shared_definitions.json#/string_array" }
+      },
+      "description": "Unique identifiers used by external sites to identify a book. Used by Open Library to link offsite.",
+      "examples": [
+        "standard_ebooks",
+        "project_gutenberg",
+        "wikisource"
+      ]
```

Expand the examples to be full

```suggestion
      "examples": [
        {
            "standard_ebooks": ["leo-tolstoy/what-is-art/aylmer-maude"]
        },
        {
            "project_gutenberg": ["64317"]
        }
      ]
```